client/test/jest/ComponentSnapshotTests/application/examine/__snapshots__/Decision.spec.js.snap

```diff
@@ -19,7 +19,7 @@ exports[`Decision.vue renders a Decision view 1`] = `
           deselectgrouplabel="Press enter to deselect group"
           deselectlabel=""
           limit="99999"
-          limittext="function(t){return\\"and \\".concat(t,\\" more\\")}"
+          limittext="function (t){return\\"and \\".concat(t,\\" more\\")}"
```

Comment:
:-/ another accidental commit.  Jeez I just assumed that anything under "test" was deliberate.